src/Uno.UI/UI/Xaml/Controls/Repeater/FlowLayoutAnchorInfo.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Linq;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public struct FlowLayoutAnchorInfo
+	{
+		internal FlowLayoutAnchorInfo(in int index, in double offset)
+		{
+			Index = index;
+			Offset = offset;
+		}
+
+		public int Index;
+
+		public double Offset;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5863680)